utils/units.js

```diff
@@ -98,3 +98,9 @@ export function parseSi(inValue, { increment = null, allowFractional = true } =
   // Unrecognized unit character
   return val;
 }
+
+export default {
```

Comment:
Doesn't really matter much here, but you should use `import { parseSi, formatSi } from '@/utils/unit'` instead of making a default export that exports everything, so that code splitting/tree shaking knows what parts of the file are actually needed/used.

Reply:
Ah I did not realize that. I'll fix it and adjust my workflow going forward.